doc-content/drools-docs/src/main/asciidoc/DMN/dmn-drd-components-ref.adoc

```diff
@@ -0,0 +1,163 @@
+[id='dmn-drd-components-ref_{context}']
+= DMN decision requirements diagram (DRD) components
+
+A decision requirements diagram (DRD) is a visual representation of your DMN model. This diagram consists of one or more decision requirements graphs (DRGs) that represent a particular domain of an overall DRD and that trace business decisions using decision nodes, business knowledge models, sources of business knowledge, input data, and decision services.
```

Comment:
Can this sentence be broken up? Such as:

"This diagram consists of one or more decision requirements graphs (DRGs) that represent a particular domain of an overall DRD. The DRGs trace business decisions using decision nodes, business knowledge models, sources of business knowledge, input data, and decision services."
or
"This diagram consists of one or more decision requirements graphs (DRGs). The DRGs represent a particular domain of an overall DRD and trace business decisions using decision nodes, business knowledge models, sources of business knowledge, input data, and decision services."